sqlmesh/core/model/definition.py

```diff
@@ -1330,9 +1326,6 @@ def _data_hash_values(self) -> t.List[str]:
             data.append(column_hash)
         return data
 
-    def __repr__(self) -> str:
-        return f"Model<name: {self.name}, seed: {self.kind.path}>"
```

Comment:
I think this is still useful

Reply:
still have it

Reply:
Can you perhaps show an example of how this looks like for seed models now?